src/app/views/query-runner/request/body/RequestBody.tsx

```diff
@@ -8,7 +8,7 @@ const RequestBody = ({ handleOnEditorChange }: any) => {
   const { dimensions: { request: { height } }, sampleQuery } = useAppSelector((state) => state);
 
   return (
-    <FocusZone id='request-body-tab'>
```

Comment:
We need this id for the global keyboard copy event that we implemented last month